src/noisepy/seis/application_modules/esyn_utils.py

```diff
@@ -0,0 +1,149 @@
+import math
+
+import numpy as np
+import pyasdf
+
+### -----
+# This scripts are aim to perform the 2-D radiative transfer equation
+# for scalar waves (Shang and Gao 1988; Sato 1993),
+# assuming of isotropic scattering and source radiation in infinite medium
+# to calculate synthesized energy densities Esyn.
+### -----
+
+
+### -----
+def read_pyasdf(sfile, ccomp):
```

Comment:
it would be better to add type annotations. Makes the code easier to use and understand
```suggestion
def read_pyasdf(sfile: str, ccomp: str) -> Tuple(float, float, np.ndarray, np.ndarray):
```

Reply:
To all functions

Reply:
Got it, thanks!